bin/tomcat-ctl

```diff
@@ -1,199 +1,246 @@
+    echo_time "Copying oracle JDBC drivers."
+    cp $TOMCAT_CONFIG_DIRECTORY/classes/* $TOMCAT_SHARE_LIB/
+
+    # Get changelogs for liquibase
+    log "Loading changelog files for liquibase."
+    echo_time "Loading changelog files for liquibase"
+
+    # create new directory to hold the files extracted from kfs-core-ua jar file
+    mkdir -p $TOMCAT_KFS_CORE_DIR
+
+    # create new directory to hold the UA changelog files
+    mkdir -p $UA_DB_CHANGELOGS_DIR
+
+    # extract files in kfs-core-ua* jar
+    cd $TOMCAT_KFS_CORE_DIR
+    /usr/bin/jar -xf $TOMCAT_KFS_DIR/WEB-INF/lib/kfs-core-ua*
+
+    # copy edu changelog files to changelogs directory
+    mv $TOMCAT_KFS_CORE_DIR/edu $UA_DB_CHANGELOGS_DIR
+
+	# run liquibase here
+    log "Running liquibase update."
+    echo_time "Running liquibase update."
+	liquibase_update_kfs.sh $APP_VERSION
+
+    log "Completed running liquibase update"
+    echo_time "Completed running liquibase update"
+
+    cp $TOMCAT_CONFIG_DIRECTORY/setenv.sh $TOMCAT_SHARE_BIN/
+    chmod +x $TOMCAT_SHARE_BIN/setenv.sh
+
+    #run logrotate on our log files
+    logrotate -f /etc/logrotate.conf
+	
+	#if rhubarb-config has not been deployed, deploy it
+    log "Checking for existing rhubarb-config deployment..."
+    echo_time "Checking for existing rhubarb-config deployment..."
+    if [ ! -d /etc/opt/kuali/rhubarb ]
+    then
+        log "No rhubarb-config deployment found, deploying rhubarb"
+        echo_time "No rhubarb-config deployment found, deploying rhubarb"
+        startdir=$(pwd)
+        service ssh start
+        mkdir -p /git-clones
+        git clone "https://github.com/ua-eas/katt-kfs-rhubarb-config" /git-clones/katt-kfs-rhubarb-config
+        cd /git-clones/katt-kfs-rhubarb-config/ && bundle install
+        cd /git-clones/katt-kfs-rhubarb-config/ && rake
+        cd /git-clones/katt-kfs-rhubarb-config/ && cap local deploy
+        cd $startdir
+        rm -R /git-clones
+        service ssh stop
+    else
+        log "rhubarb-config already deployed, no action required"
+        echo_time "rhubarb-config already deployed, no action required"
+    fi
+    #if rhubarb has not been deployed, deploy it
+    log "Checking for existing rhubarb deployment..."
+    echo_time "Checking for existing rhubarb deployment..."
+    if [ ! -d /opt/kuali/rhubarb ]
+    then
+        log "No rhubarb deployment found, deploying rhubarb"
+        echo_time "No rhubarb deployment found, deploying rhubarb"
+        startdir=$(pwd)
+        service ssh start
+        mkdir -p /git-clones
+        git clone "https://github.com/ua-eas/katt-kfs-rhubarb" /git-clones/katt-kfs-rhubarb
+        cd /git-clones/katt-kfs-rhubarb/ && bundle install
+        mkdir -p /git-clones/katt-kfs-rhubarb/batch_home/logs
+        mkdir -p /git-clones/katt-kfs-rhubarb/spec/live/uaf-stg
+        export BATCH_HOME=/git-clones/katt-kfs-rhubarb/batch_home
+        export RHUBARB_ENV=DEV
+        cd /git-clones/katt-kfs-rhubarb/ && rake

```

Comment:
Same thought about running the tests during startup. I think we can remove this part and do it in a Jenkins job whenever Rhubarb code is committed.